src/Google.Pubsub.V1/PublisherClient.cs

```diff
@@ -448,12 +515,20 @@ public sealed partial class PublisherClientImpl : PublisherClient
             );
 
         private readonly ClientHelper _clientHelper;
+        private readonly ApiCallableAsync<Topic, Topic> _createTopicAsync;
+        private readonly ApiCallableSync<Topic, Topic> _createTopicSync;
+        private readonly ApiCallableAsync<ListTopicsRequest, ListTopicsResponse> _listTopicsAsync;
+        private readonly ApiCallableSync<ListTopicsRequest, ListTopicsResponse> _listTopicsSync;
 
         public PublisherClientImpl(Publisher.IPublisherClient grpcClient, PublisherSettings settings)
         {
             this.GrpcClient = grpcClient;
             PublisherSettings effectiveSettings = settings ?? PublisherSettings.GetDefault();
             _clientHelper = new ClientHelper(effectiveSettings);
+            _createTopicAsync = _clientHelper.BuildApiCallableAsync(GrpcClient.CreateTopicAsync, effectiveSettings.CreateTopic_Retry);
```

Comment:
BCL has AsyncCallback delegate; but it's not really relevant as it's not an async delegate, it's a delegate to use to pass a callback function.
I can't find others right now.
I think I'd rather keep the Async as a suffix unless we have a good reason not to (eg prior art)